test/index.test.js

```diff
@@ -505,6 +505,27 @@ describe('Segment.io', function() {
             analytics.stub(segment, 'enqueue');
           });
 
+          it('identify should not ultimately call getCachedCrossDomainId if crossDomainAnalytics is not enabled', function() {
+            var wasCalled = false;
+            var oldCrossDomainCheck = segment.isCrossDomainAnalyticsEnabled;
+            segment.isCrossDomainAnalyticsEnabled = function() {
```

Comment:
I don't think `isCrossDomainAnalyticsEnabled` needs to be mocked, it can be disabled with `segment.options.crossDomainIdServers = [];` (example [here]( https://github.com/segment-integrations/analytics.js-integration-segmentio/blob/2e202b92e7baa508174a43b4468af1a75c763653/test/index.test.js#L366)).

Reply:
done!